### ISSUE TITLE
Customize TextStyle

### ISSUE THREAD

**Issue by christian-froh** [flutter-stripe/flutter_stripe#1869]
It would be nice to customize the textStyle or use the themeData.

**Comment by remonh87** [comment]
Can you provide some additional context to this?  What are you missing and for which component? Our widgets are native rendered and maintained by stripe so there is little we can do. Themedata is for sure not possible

**Comment by christian-froh** [comment]
Yeah you are right. According to the documentation the font is customizable https://docs.stripe.com/elements/appearance-api?platform=ios#fonts-ios
Would that be possible?

**Comment by remonh87** [comment]
Yes but then you must add it natively. I think it could work but we need to do some testing on our side and add font config

**Comment by remonh87** [comment]
Currently not on the roadmap
